Revise Swift entries (#929)
Codecademy/docs: content/swift/concepts/comments/comments.md

---
Title: 'Comments'

> Description: 'Comments in Swift represent text that explains the syntax of a piece of raw code. In Swift there single-line comments and multi-line comments. '

Reviewer: ```suggestion
Description: 'Comments in Swift represent text that explains the raw code. In Swift there single-line comments and multi-line comments. '
```

  - 'paths/build-ios-apps-with-swiftui'
---

> Comments in Swift represent text that is useful to the programmer or developement team. Think of them as notes or reminders of important concepts or ideas that require explaination above the syntax of the raw code.

Reviewer: ```suggestion
**Comments** in Swift represent text that is useful to the programmer or development team. Think of them as notes or reminders of important concepts or ideas that require explanation beyond the syntax of the raw code.
```

> **Comments** in Swift represent text that is useful to the programmer or development team. Think of them as notes or reminders of important concepts or ideas that require explanation beyond the syntax of the raw code.
> 
> In Swift, there are two kinds of comments: single-line comments and multi-line comments. These comments will be completely ignored by the compiler at compile-time.

Reviewer 2: ```suggestion
**Comments** in Swift represent text that is ignored by the compiler at-compile time, but is useful to the programmer or development team. They are ideal for documenting important concepts or ideas that require explanation beyond the syntax of the raw code. Both single-line and multi-line comments are supported in Swift.
```
